src/03-modality-agnostic-files.md

```diff
@@ -251,6 +251,71 @@ to date of birth.
 }
 ```
 
+## Samples file
+
+Template:
+
+```Text
+samples.tsv
+samples.json
+```
+
+The purpose of this RECOMMENDED file is to describe properties of samples
+when the `sample-<label>` key/value pair is used in filenames.
+If this file exists, it MUST contain the three following columns:
+
+-   `sample_id`: MUST consist of `sample-<label>` values identifying one row
+    for each sample
+
+-   `participant_id`: MUST consist of `sub-<label>`
+
+-   `sample_type`: MUST consist of sample type values (for example: tissue,
+    primary cell or cell-free sample) from [ENCODE Biosample Type](https://www.encodeproject.org/profiles/biosample_type)
+
+Other optional columns MAY be used to describe the samples.
+Each sample MUST be described by one and only one row.
+
+Commonly used *optional* columns in `samples.tsv` files are `pathology` and
+`derived_from`. We RECOMMEND to make use of these columns, and in case that
+you do use them, we RECOMMEND to use the following values for them:
+
+-   `pathology`: string value describing the pathology of the sample or type of control.
+    When different from `healthy`, pathology SHOULD be specified in `samples.tsv`.
+    The pathology MAY instead be specified in [Sessions files](06-longitudinal-and-multi-site-studies.md#sessions-file)
+    in case it changes over time.
+
+-   `derived_from`: `sample-<label>` key/value pair from which a sample is derived from,
+    for example a slice of tissue (`sample-02`) derived from a block of tissue (`sample-01`),
+    as illustrated in the example below.
```

Comment:
this makes me think that a samples.tsv could refer to entities that are not in the filenames. which would be fine. but we may want to illustrate or clarify.

Reply:
I’m sorry, I don’t understand what you mean, could you give me an example. Thanks!

Reply:
imagine you submit data from a slab (slab-01), which came from a block (sample-01). both are samples, and one may want to provide properties for both, even though the data itself is about the slab. so:

```
sample-01, tissue, sub-01
slab-01, slice, sample-01
```

Reply:
In the current proposal, if `slab-01` and `sample-01` are both samples, they are required to be identified with a `sample-<label>`. This was the idea behind `sample` to avoid adding several entities to the specs like `block`, `slice`, `slab`, etc. So any of these should be identified as a `sample`. From your example, that would give something like this, where the `id` is any alphanumeric string:

|sample_id|participant_id   |sample_type|derived_from|
|---------|-----------------|-----------|------------|
|sample-01|sub-01         |tissue     |n/a         |
|sample-slab01|sub-01         |tissue     |sample-01|

In the first line: because every sample is always associated with a subject (`ex: sub-01_sample-01_SEM.ome.tif`), `sub-01` goes in `participant_id` and not in `derived_from`.
In the second line: `slice` is not one of the `sample_type` accepted in ENCODE biosample, they would both be `tissue` in this case.

Let me know if I interpreted correctly your comment. Thanks!

Reply:
thank you @mariehbourget - this looks great.